shtub.sh

```diff
@@ -223,16 +223,28 @@ _stub::methods::returns() {
 # Sets a stub to error by printing the given string to stderr and returning the
 # given status code.
 # @param $1 name Name of the command to stub
```

Comment:
I think it will be okay for now since it is consistent. I am going to look into automatic doc generators for bash scripts and see if there is a particular style we should be using here.